spinner.c

```diff
@@ -175,7 +175,28 @@ char* char_sets[][MAX_CHARS] = {
       "▐    ⠂   ▌", "▐    ⠠   ▌", "▐     ⡀  ▌", "▐     ⠠  ▌", "▐      ⠂ ▌", "▐      ⠈ ▌", "▐       ⠂▌", "▐       ⠠▌",
       "▐       ⡀▌", "▐      ⠠ ▌", "▐      ⠂ ▌", "▐     ⠈  ▌", "▐     ⠂  ▌", "▐    ⠠   ▌", "▐    ⡀   ▌", "▐   ⠠    ▌",
       "▐   ⠂    ▌", "▐  ⠈     ▌", "▐  ⠂     ▌", "▐ ⠠      ▌", "▐ ⡀      ▌", "▐⠠       ▌" },
-    { "?", "¿" }
+    { "?", "¿" },
+    { "⢹", "⢺", "⢼", "⣸", "⣇", "⡧", "⡗", "⡏" },
+    { "⢄", "⢂", "⢁", "⡁", "⡈", "⡐", "⡠" },
+    { ".  ", ".. ", "...", " ..", "  .", "   " },
+    { ".", "o", "O", "°", "O", "o", "." },
+    { "▓", "▒", "░" },
+    { "▌", "▀", "▐", "▄" },
+    { "⊶", "⊷" },
+    { "▪", "▫" },
+    { "□", "■" },
+    { "▮", "▯" },
+    { "-", "=", "≡" },
+    { "d", "q", "p", "b" },
+    { "∙∙∙", "●∙∙", "∙●∙", "∙∙●", "∙∙∙" },
+    { "🌑 ", "🌒 ", "🌓 ", "🌔 ", "🌕 ", "🌖 ", "🌗 ", "🌘 " },
+    { "☗", "☖" },
+    { "⧇", "⧆" },
+    { "◉", "◎" },
+    { "㊂", "㊀", "㊁" },
+    { "⦾", "⦿" },
+    { "ဝ", "၀" },
+    { "▌", "▀", "▐▄" }
```

Comment:
Is the comma after the last index optional?

Reply:
Yup!